Network Analyzer - Commands/World Commands (#1174)
spatialos/gdk-for-unity: CHANGELOG.md

## Unreleased

### Added

> - Add Commands and World Commands tabs for Network Analyzer. [#1174](https://github.com/spatialos/gdk-for-unity/pull/1174)

Reviewer: ```suggestion
- Added commands and world commands tabs to the Network Analyzer window. [#1174](https://github.com/spatialos/gdk-for-unity/pull/1174)
```
